apps/core/app/[locale]/(default)/(faceted)/brand/[slug]/static/page.tsx

```diff
@@ -6,9 +8,11 @@ export default BrandPage;
 
 export async function generateStaticParams() {
   const brands = await getBrands();
+  const locale = await getLocale();
```

Comment:
How is this working with static generated pages? Is it generating a static page for every supported locale? That doesn't *look* like what's happening based on what I'm reading here, but could be misunderstanding.

Reply:
You're right. I've moved it to Home page. Guess, it's a right place.